build.gradle

```
@@ -129,7 +129,7 @@ allprojects {
     java {
       // This path needs to be relative to each project
       target '**/*.java'
-      targetExclude '**/src/reference-test/**', '**/src/main/generated/**', '**/src/test/generated/**', '**/src/jmh/generated/**'
+      targetExclude '**/src/reference-test/**', '**/src/main/generated/**', '**/src/test/generated/**', '**/src/jmh/generated/**', '.idea/**'
```

Rather than one-by-one exclusions we should have a more focused inclusion.
```suggestion
      target 'src/**/*.java'
      targetExclude 'src/reference-test/**', 'src/main/generated/**', 'src/test/generated/**', 'src/jmh/generated/**'
```

---

You are right. That is much better. I've changed it.